specification/machinelearningservices/data-plane/Microsoft.MachineLearningServices/preview/2019-05-01/modelManagement.json

```diff
@@ -0,0 +1,4613 @@
+            "description": "The payload that is used to patch an Image.",
+            "required": true,
+            "schema": {
+              "uniqueItems": false,
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/JsonPatchOperation"
+              }
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The Image patch was successful.",
+            "schema": {
+              "$ref": "#/definitions/ImageResponseBase"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ModelErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/modelmanagement/v1.0/subscriptions/{subscriptionId}/resourceGroups/{resourceGroup}/providers/Microsoft.MachineLearningServices/workspaces/{workspace}/images": {
+      "get": {
+        "tags": [
+          "Image"
+        ],
+        "summary": "Query the list of Images in a workspace.",
+        "description": "If no filter is passed, the query lists all the Images in the given workspace. The returned list is paginated and the count of items in each page is an optional parameter.",
+        "operationId": "Images_ListQuery",
+        "consumes": [],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupParameter"
+          },
+          {
+            "$ref": "#/parameters/workspaceParameter"
+          },
+          {
+            "name": "name",
+            "in": "query",
+            "description": "The object name.",
+            "required": false,
+            "type": "string"
+          },
+          {
+            "name": "tag",
+            "in": "query",
+            "description": "The object tag.",
+            "required": false,
+            "type": "string"
+          },
+          {
+            "name": "description",
+            "in": "query",
+            "description": "The description of the object.",
+            "required": false,
+            "type": "string"
+          },
+          {
+            "name": "modelId",
+            "in": "query",
+            "description": "The Model Id.",
+            "required": false,
+            "type": "string"
+          },
+          {
+            "name": "modelName",
+            "in": "query",
+            "description": "The Model name.",
+            "required": false,
+            "type": "string"
+          },
+          {
+            "name": "count",
+            "in": "query",
+            "description": "The number of items to retrieve in a page.",
+            "required": false,
+            "type": "string"
+          },
+          {
+            "name": "$skipToken",
```

Comment:
Where does the $skipToken value come from?

Reply:
Need a better description and example.

Reply:
Fix underway.

Reply:
Fixed